config.py

```diff
@@ -6,7 +6,7 @@ class AmoStorageConfig(object):
 
     # SQLite Configurations
     SQLALCHEMY_TRACK_MODIFICATIONS = False
-    SQLALCHEMY_DATABASE_URI = 'sqlite:///tmp/amo_storage.db'
+    SQLALCHEMY_DATABASE_URI = 'sqlite:////tmp/amo_storage.db'
```

Comment:
Make it configurable

Reply:
It's already configurable. User can change DATABASE URI on config.py as above. I have no idea what you're talking about.

Reply:
Hmmm... I didn't imagine this file is a user configuration file. I thought this is just for default config values. I expected more popular type for a user configuration file, e.g. ini, yaml, json, config, and so on. Are you planning to keep this format? In that case, a user surely needs a configuration guide to write a *class* to config something. In real env, an administrator may not know anything about python grammar.

Reply:
We gonna add configuration guide. This config.py is pythonic style to handle configurations(using built-in data structure). If you want external file like .ini or .json, we will reflect it.

Reply:
It's much simpler to use ini file for users. Please make a new PR for the job. I'll merge this in this state.